utils/proxy.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Get proxy configuration from environment variables
+ */
+export function getProxyConfig() {
```

Comment:
is this used in other places than nuxt.config.ts now?

Reply:
used in the other function `getBackendUrl`, and I need it to get the metadata (contract_address, chain), in the server

Reply:
is that the only way can't we access it any other way?

Reply:
Hmm i'm not sure, so since it's called from nuxt server, it needs to call the backend endpoint directly (http://localhost:8000) directly, do you have any suggestion?